Stripe/STPPaymentCardTextField.m

```diff
@@ -476,24 +633,155 @@ - (void)setText:(NSString *)text inField:(STPCardFieldType)field {
         case STPCardFieldTypeCVC:
             textField = self.cvcField;
             break;
+        case STPCardFieldTypePostalCode:
+            textField = self.postalCodeField;
+            break;
     }
     textField.text = nonNilText;
 }
 
+- (CGFloat)numberFieldFullWidth {
+    // Current longest possible pan is 16 digits which our standard sample fits
+    if ([self.viewModel validationStateForField:STPCardFieldTypeNumber] == STPCardValidationStateValid) {
+        return [self widthForCardNumber:self.viewModel.cardNumber];
+    }
+    else {
+        return MAX([self widthForCardNumber:self.viewModel.cardNumber],
+                   [self widthForCardNumber:self.viewModel.defaultPlaceholder]);
+    }
+}
+
+- (CGFloat)numberFieldCompressedWidth {
+    // Current longest possible longest pan fragment is 5 characters
+    return [self widthForText:@"88888"];
```

Comment:
I think this non-localized string will trigger an analyze warning once you update this branch to `master`.